ports/psoc6/help.c

```diff
@@ -33,16 +33,17 @@ const char psoc6_help_text[] =
     "are in the 'psoc6' module.\n"
     "\n"
     "Quick overview of some objects:\n"
-    "  machine.Pin(pin) -- get a pin, eg machine.Pin(0)\n"
+    "  machine.Pin(pin) -- get a pin, eg machine.Pin('P9_0')\n"
     "  machine.Pin(pin, m, [p]) -- get a pin and configure it for IO mode m, pull mode p\n"
     "    methods: init(..), value([v]), high(), low(), irq(handler)\n"
-    "  machine.I2C(id) -- create an I2C object (id=0,1)\n"
+    "\n"
+    "    Pin IO modes are: Pin.IN, Pin.OUT\n"
+    "    Pin pull modes are: Pin.PULL_UP, Pin.PULL_DOWN\n"
+    "\n"
+    "  machine.I2C(scl, sda) -- create an I2C object\n"
     "    methods: readfrom(addr, buf, stop=True), writeto(addr, buf, stop=True)\n"
```

Comment:
I know it's not added in this PR, but why do we have just PIN and I2C here? How is "some" defined here?

Reply:
hehe. Yep, I agree. 
Did not want to open more boxes now, and decided to simply fixed what was wrong in our API. But yes, which "some" are to be added here?
So I decided to do nothing about it now. We can also remove that?
What you @all think?

Reply:
Let's remove!

Reply:
Yes Let's remove that

Reply:
I will :)